`suggest-promoted-replacement` crashes on Ubuntu 14.04

```
$ lsb_release -d
Description:	Ubuntu 14.04.5 LTS
```

```
$ orchestrator -c suggest-promoted-replacement -i <cluster>
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x85d147]

goroutine 1 [running]:
github.com/github/orchestrator/go/inst.ReadInstance(0x0, 0xc4201c3298, 0x1dcefca2, 0xb898438262f923a9, 0xc400000001)
	/home/shlomi-noach/go/src/github.com/github/orchestrator/go/inst/instance_dao.go:1032 +0x47
github.com/github/orchestrator/go/app.validateInstanceIsFound(0x0, 0x1c)
	/home/shlomi-noach/go/src/github.com/github/orchestrator/go/app/cli.go:122 +0x2f
github.com/github/orchestrator/go/app.Cli(0x7ffc25a076a2, 0x1c, 0x0, 0xc420198c50, 0x9, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/shlomi-noach/go/src/github.com/github/orchestrator/go/app/cli.go:1398 +0xbb9b
github.com/github/orchestrator/go/app.CliWrapper(0x7ffc25a076a2, 0x1c, 0x0, 0x7ffc25a076c2, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/shlomi-noach/go/src/github.com/github/orchestrator/go/app/cli.go:149 +0x20d
main.main()
	/home/shlomi-noach/go/src/github.com/github/orchestrator/go/cmd/orchestrator/main.go:158 +0xdb9
```

--- a/go/app/cli.go
+++ b/go/app/cli.go
@@ -1393,15 +1393,6 @@ func Cli(command string, strict bool, instance string, destination string, owner
 			}
 			fmt.Println(fmt.Sprintf("%d recoveries acknowldged", countRecoveries))
 		}
-	case registerCliCommand("suggest-promoted-replacement", "Recovery", `Information command suggesting the best would-be replacement to master of given cluster`):
-		{
-			destination := validateInstanceIsFound(destinationKey)
-			replacement, _, err := logic.SuggestReplacementForPromotedReplica(&logic.TopologyRecovery{}, instanceKey, destination, nil)
-			if err != nil {
-				log.Fatale(err)
-			}
-			fmt.Println(replacement.Key.DisplayString())
-		}
 	// Instance meta
 	case registerCliCommand("register-candidate", "Instance, meta", `Indicate that a specific instance is a preferred candidate for master promotion`):
 		{
@@ -1531,6 +1522,15 @@ func Cli(command string, strict bool, instance string, destination string, owner
 			config.RuntimeCLIFlags.ConfiguredVersion = ""
 			inst.ReadClusters()
 			fmt.Println("Redeployed internal db")
+		}
+	case registerCliCommand("internal-suggest-promoted-replacement", "Internal", `Internal only, used to test promotion logic in CI`):
+		{
+			destination := validateInstanceIsFound(destinationKey)
+			replacement, _, err := logic.SuggestReplacementForPromotedReplica(&logic.TopologyRecovery{}, instanceKey, destination, nil)
+			if err != nil {
+				log.Fatale(err)
+			}
+			fmt.Println(replacement.Key.DisplayString())
 		}
 	case registerCliCommand("custom-command", "Agent", "Execute a custom command on the agent as defined in the agent conf"):
 		{

--- a/tests/integration/suggest-promoted-replacement-candidate-different-dc/extra_args
+++ b/tests/integration/suggest-promoted-replacement-candidate-different-dc/extra_args
@@ -1,1 +1,1 @@
--c suggest-promoted-replacement -i testhost:22293 -d testhost:22294
+-c internal-suggest-promoted-replacement -i testhost:22293 -d testhost:22294

--- a/tests/integration/suggest-promoted-replacement-candidate-same-dc/extra_args
+++ b/tests/integration/suggest-promoted-replacement-candidate-same-dc/extra_args
@@ -1,1 +1,1 @@
--c suggest-promoted-replacement -i testhost:22293 -d testhost:22294
+-c internal-suggest-promoted-replacement -i testhost:22293 -d testhost:22294

--- a/tests/integration/suggest-promoted-replacement-different-dc/extra_args
+++ b/tests/integration/suggest-promoted-replacement-different-dc/extra_args
@@ -1,1 +1,1 @@
--c suggest-promoted-replacement -i testhost:22293 -d testhost:22294
+-c internal-suggest-promoted-replacement -i testhost:22293 -d testhost:22294

--- a/tests/integration/suggest-promoted-replacement-ignore-no-binlog/extra_args
+++ b/tests/integration/suggest-promoted-replacement-ignore-no-binlog/extra_args
@@ -1,1 +1,1 @@
--c suggest-promoted-replacement -i testhost:22293 -d testhost:22294
+-c internal-suggest-promoted-replacement -i testhost:22293 -d testhost:22294

--- a/tests/integration/suggest-promoted-replacement-is-candidate/extra_args
+++ b/tests/integration/suggest-promoted-replacement-is-candidate/extra_args
@@ -1,1 +1,1 @@
--c suggest-promoted-replacement -i testhost:22293 -d testhost:22294
+-c internal-suggest-promoted-replacement -i testhost:22293 -d testhost:22294

--- a/tests/integration/suggest-promoted-replacement-multiple-candidates-2/extra_args
+++ b/tests/integration/suggest-promoted-replacement-multiple-candidates-2/extra_args
@@ -1,1 +1,1 @@
--c suggest-promoted-replacement -i testhost:22293 -d testhost:22294
+-c internal-suggest-promoted-replacement -i testhost:22293 -d testhost:22294

--- a/tests/integration/suggest-promoted-replacement-multiple-candidates/extra_args
+++ b/tests/integration/suggest-promoted-replacement-multiple-candidates/extra_args
@@ -1,1 +1,1 @@
--c suggest-promoted-replacement -i testhost:22293 -d testhost:22294
+-c internal-suggest-promoted-replacement -i testhost:22293 -d testhost:22294

--- a/tests/integration/suggest-promoted-replacement-prefer-not-2/extra_args
+++ b/tests/integration/suggest-promoted-replacement-prefer-not-2/extra_args
@@ -1,1 +1,1 @@
--c suggest-promoted-replacement -i testhost:22293 -d testhost:22294
+-c internal-suggest-promoted-replacement -i testhost:22293 -d testhost:22294

--- a/tests/integration/suggest-promoted-replacement-prefer-not/extra_args
+++ b/tests/integration/suggest-promoted-replacement-prefer-not/extra_args
@@ -1,1 +1,1 @@
--c suggest-promoted-replacement -i testhost:22293 -d testhost:22294
+-c internal-suggest-promoted-replacement -i testhost:22293 -d testhost:22294

--- a/tests/integration/suggest-promoted-replacement-trivial/extra_args
+++ b/tests/integration/suggest-promoted-replacement-trivial/extra_args
@@ -1,1 +1,1 @@
--c suggest-promoted-replacement -i testhost:22293 -d testhost:22294
+-c internal-suggest-promoted-replacement -i testhost:22293 -d testhost:22294

